Cargo.toml

```diff
@@ -121,7 +121,12 @@ tracing-error = "0.2.0"
 tracing-log = "0.2.0"
 tracing-subscriber = { version = "0.3.18", features = ["env-filter"] }
 url = { version = "2.5.0", features = ["serde"] }
-reqwest = { version = "0.11.27", features = ["json", "blocking", "gzip"] }
+reqwest = { version = "0.11.27", default-features = false, features = [
+  "json",
+  "blocking",
+  "gzip",
+  "rustls-tls",
+] }
```

Comment:
I removed the default features for reqwest, and added rusttls-tls, but for some reason, `native-tls` is still in the `Cargo.lock`. 

Might need someone's help for why that's happening.

https://docs.rs/crate/reqwest/0.11.27/features

Reply:
May need to enable rustls feature for reqwest-middleware as well?

https://github.com/TrueLayer/reqwest-middleware/blob/4ce811911bf841db6e7d91efd65ba7e5e40a00f6/reqwest-middleware/Cargo.toml#L18

Reply:
Hmm... there are two versions of reqwest, of which 0.12.4 does not depend on openssl.

0.11.27 is a dependency of activitypub-federation, so this repository needs to be updated and enable rustls-tls feature.

Reply:
https://github.com/LemmyNet/activitypub-federation-rust/pull/118

Reply:
K, I'll do that for the middleware also.

Reply:
Ive published version 0.5.8 of activitypub-federation which doesnt use openssl anymore.